src/components/radio/radio-button.scss

```diff
@@ -0,0 +1,139 @@
+@import "default-theme";
+
+$radio-width: 20px !default;
+$radio-height: $radio-width !default;
+
+// Top-level container for entire button.
+md-radio-button {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px;
```

Comment:
How possible is it to move things like this padding into variables that could be overridden by users in the name of density (which I know the designers have been thinking about but not yet really addressing in the spec)?